spack/packages/dla-future/package.py

```diff
@@ -68,6 +68,19 @@ class DlaFuture(CMakePackage, CudaPackage, ROCmPackage):
 
     conflicts("+cuda", when="+rocm")
 
+    with when("+rocm"):
+        for val in ROCmPackage.amdgpu_targets:
+            depends_on("pika amdgpu_target={0}".format(val),
+                when="amdgpu_target={0}".format(val))
+            depends_on("hipblas amdgpu_target={0}".format(val),
+                when="amdgpu_target={0}".format(val))
+            depends_on("rocsolver amdgpu_target={0}".format(val),
+                when="amdgpu_target={0}".format(val))
+            depends_on("rocblas amdgpu_target={0}".format(val),
+                when="amdgpu_target={0}".format(val))
+            depends_on("umpire amdgpu_target={0}".format(val),
+                when="amdgpu_target={0}".format(val))
+
```

Comment:
I remember asking about this but I don't remember the answer: What will spack do if `dla-future` has `amdgpu_target=gfx90a` and e.g. `rocblas` has been installed with either `amdgpu_target=auto` or `amdgpu_target=gfx90a,more-targets...`?

Reply:
Thanks! No I think if it works with one dependency it should work the same for any.